pkg/controllers/organization/rbac_controller.go

```diff
@@ -31,92 +26,7 @@ const (
 	pluginAdmin
 )
 
-// RBACReconciler reconciles an Organization object and manages RBAC permissions based on organization and team membership.
-type RBACReconciler struct {
-	client.Client
-	recorder record.EventRecorder
-}
-
-//+kubebuilder:rbac:groups=greenhouse.sap,resources=organizations,verbs=get;list;watch;create;update;patch
-//+kubebuilder:rbac:groups=greenhouse.sap,resources=organizations/status,verbs=get;update;patch
-//+kubebuilder:rbac:groups=greenhouse.sap,resources=organizations/finalizers,verbs=update
-//+kubebuilder:rbac:groups="rbac.authorization.k8s.io",resources=clusterroles;clusterrolebindings;roles;rolebindings,verbs=get;list;watch;create;update;patch
-//+kubebuilder:rbac:groups="",resources=events,verbs=get;list;watch;create;update;patch;delete
-
-// SetupWithManager sets up the controller with the Manager.
-func (r *RBACReconciler) SetupWithManager(name string, mgr ctrl.Manager) error {
-	r.Client = mgr.GetClient()
-	r.recorder = mgr.GetEventRecorderFor(name)
-	return ctrl.NewControllerManagedBy(mgr).
-		Named(name).
-		For(&greenhouseapisv1alpha1.Organization{}).
-		Owns(&rbacv1.Role{}).
-		Owns(&rbacv1.RoleBinding{}).
-		Owns(&rbacv1.ClusterRole{}).
-		Owns(&rbacv1.ClusterRoleBinding{}).
```

Comment:
These should move the the OrganizationReconciler's `SetupWithManager(..)`

Reply:
Done